scaaml/io/dataset.py

```diff
@@ -1058,21 +1059,21 @@ def cleanup_shards(dataset_path):
 
         Example use:
           Dataset.cleanup_shards(
-              dataset_path='/mnt/storage/chipwhisperer/test_cleanup')
+              dataset_path="/mnt/storage/chipwhisperer/test_cleanup")
         """
         fpath = Dataset._get_config_path(dataset_path)
 
         # Save the old config.
-        save_path = Path(f'{str(fpath)}.sav.{time()}.json')
+        save_path = Path(f"{str(fpath)}.sav.{time()}.json")
         assert not save_path.exists()  # Do not overwrite.
         save_path.write_text(fpath.read_text())
-        cprint("Saving old config to %s" % save_path, 'cyan')
+        cprint(f"Saving old config to {save_path}", "cyan")
```

Comment:
Maybe also we may want to make sure that color printing give a consistant UX throughout our code and instead of writing the color, on each `cprint()` call, we could have a bunch of helpers that wrap them (e.g. `cprint_info()`, `cprint_warning()`, `cprint_error()`, etc.).